Requirements/cross-chain-transfer.md

```diff
@@ -2,63 +2,65 @@
 
-Finally, as chain B progresses and the transfer's maturity passes the maximum reorg length, all federation nodes monitoring the B chain can update the status of the corresponding session in their respectives stores. Once the status of the session is marked complete, no more work will be required for this session.
+Finally, as chain B progresses and the transfer's maturity passes the maximum reorg length, all federation nodes monitoring the B chain can update the status of the corresponding session in their respective stores. Once the status of the session is marked complete, no more work will be required for this session.
 
 ![The cross chain transfer matures on targeted chain](../assets/cross-chain-transfers/happy-path-4.svg)
 
 ## 2) Preferred leader offline
 
 ### A cross chain transfer is triggered while leader is offline
 
-Similar to the previous case, deposit transactions to the federation's multisig on chain A have triggered a cross chain transfer. Here however, the prefered leader happens to be offline. All other nodes carry on as usual, creating and exchanging partially signed transactions. However, no one is here to broadcast it as FM<sub>2</sub> is offline.
+Similar to the previous case, deposit transactions to the federation's multisig on chain A have triggered a cross chain transfer. Here however, the preferred leader happens to be offline. All other nodes carry on as usual, creating and exchanging partially signed transactions. However, no one is here to broadcast it as FM<sub>2</sub> is offline.
 
 ![A cross chain transfer is triggered while leader is offline](../assets/cross-chain-transfers/leader-offline-1.svg)
 
 ### The cross chain is handled one block later by next leader
 
-As time passes, a new block appears on chain A, and a new leader is elected (FM<sub>3</sub>). This new leader can now broadcast the transaction and we are back on the normal path (cf 1.).  
+As time passes, a new block appears on chain A, and a new leader is elected (FM<sub>3</sub>). This new leader can now broadcast the transaction and we are back on the normal path (cf 1.).
 
-As described earlier every new leader must first check for previously unprocessed transfer and process those first before attending to the transfers on current block, if a few leaders are offline this can create a backlog of transfers.  
-If a leader finds a previously unprocessed transfer and they dont have enough signatures to broadcast they must do nothing and wait for the other members to send over signatures.  
-If not enough members are online to reach the minimum number of signature required by the multisig, then transfers will halt untill enough members come back online and broadcast signatures.
+As described earlier, every new leader must first check for previously unprocessed transfers and process those first, before attending to the transfers on the current block. If a few leaders are offline, this can create a backlog of transfers.
```

Comment:
the 2 spaces are added at the end of lines as a Markdown way to add a breakline. Did you really want to remove them? I liked it, but up to you ;)